sirmordred/task_enrich.py

```diff
@@ -223,6 +237,51 @@ def __autorefresh(self):
         else:
             logger.debug("No ids to be refreshed found")
 
+        # Update corresponding autorefresh date
+        if studies:
+            self.last_autorefresh_studies = next_autorefresh
+        else:
+            self.last_autorefresh = next_autorefresh
+
+    def __autorefresh_studies(self, cfg):
+        """Execute autorefresh for areas of code study if configured"""
+
+        if 'studies' not in self.conf[self.backend_section] or \
+                'enrich_areas_of_code:git' not in self.conf[self.backend_section]['studies']:
+            logger.debug("Not doing autorefresh for studies, Areas of Code study is not active.")
+            return
+
+        aoc_index = None
+        if 'out_index' in self.conf['enrich_areas_of_code:git']:
+            aoc_index = self.conf['enrich_areas_of_code:git']['out_index']
+
+        # Not put this into an else statement to avoid problems if conf field
+        # exists but is empty
+        if not aoc_index:
```

Comment:
If it exists but it was empty it will enter on this block. What's the difference then?